pages/docs/environments/gitpod.mdx

```diff
@@ -59,7 +118,9 @@ Now we can update our profile. To do this, create a new branch with the followin
 git checkout -b update-YOUR_PROFILE_NAME
 ```
 
-Now we are ready to make some modification to our profile. Find the file with your account name in the `data` folder, and modify it as you want.
+<Alert type="info" message={ <> NOTE: To create your profile on LinkFree, please follow the [Quickstart guide](https://linkfree.io/docs/quickstart). </> } />
+
+Now that we have our profile, we are ready to make some modification to our profile. Find the file with your account name in the `data` folder, and modify it as you want.
 
 For example, you can copy and past the following code
```

Comment:
[Proofreading]
"For example, you can copy and paste the following code:"

Reply:
It seems perfect! :)

Reply:
No, @JenIsCoding, I mean you have to make at edit lol 😄